lib/Dialect/Torch/IR/TorchOps.cpp

```diff
@@ -1606,6 +1606,97 @@ OpFoldResult ValueTensorLiteralOp::fold(FoldAdaptor adaptor) {
   return getValueAttr();
 }
 
+void ValueTensorLiteralOp::getCanonicalizationPatterns(
+    RewritePatternSet &patterns, MLIRContext *context) {
+  patterns.add(+[](ValueTensorLiteralOp constOp, PatternRewriter &rewriter) {
+    if (!constOp->hasOneUse())
+      return failure();
+
+    OpOperand *use = constOp.getResult().use_begin().getOperand();
+    auto op = dyn_cast<OperatorOp>(use->getOwner());
+    if (!op)
+      return failure();
+
+    if (use->getOperandNumber() != 1)
+      return failure();
+
+    if (op.getName().str() != "brevitas.matmul_rhs_group_quant") {
```

Comment:
Instead of using torch.operator here, can we add `brevitas.matmul_rhs_group_quant` to TorchOps.td?